projects/ecoindex_cli/README.md

```diff
@@ -102,6 +102,35 @@ There are 2 url(s), do you want to process? [Y/n]:
 
 </details>
 
+### Make a website analysis based on the website's sitemap
+
+The cli allows for the analysis of a designated website using its sitemap. By entering the sitemap URL, the app identifies and analyzes all pages within the website. ⚠️ This can process for a very long time! **Use it at your own risks!**
```

Comment:
**🎯 suggestion(non-blocking):** Add a comment to explain that if we set `recursive` option too, it will be ignored..